src/pages/ChainsViewPage.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react'
+import 'feather-icons'
+import axios from 'axios';
+
+import { TYPE } from '../Theme'
+import Panel from '../components/Panel'
+import { useAllTokenData } from '../contexts/TokenData'
+import { PageWrapper, FullWrapper, ContentWrapper } from '../components'
+import Row, { RowBetween } from '../components/Row'
+import { AutoColumn } from '../components/Column'
+import ChainsChart from '../components/ChainsChart'
+import LocalLoader from '../components/LocalLoader'
+
+
+// import React from 'react'
+
+const ChainsView = () => {
+    let allTokens = useAllTokenData();
+
+    const allChains = Object.values(allTokens).map((chain) => {
+        return chain.chain
+    });
+
+    const getUnique = arr => {
+        return arr.filter((value, index, self) => {
+            return self.indexOf(value) === index;
+        });
+    };
+
+    const chainsUnique = getUnique(allChains);
+
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(true);
+
+    axios.defaults.baseURL = 'https://api.llama.fi/charts/';
```

Comment:
Does this affect other instances of axios? if so it may be better to not do this since it could be unexpected in other parts of the app